src/NHibernate.Test/Linq/ODataTests.cs

```diff
@@ -75,6 +75,21 @@ public void BasePropertyFilter(string queryString, int expectedRows)
 			Assert.That(results, Has.Count.EqualTo(expectedRows));
 		}
 
+		//GH-2362
+		[TestCase("$filter=CustomerId le 'ANATR'",2 )]
+		[TestCase("$filter=startswith(CustomerId, 'ANATR')", 1)]
+		[TestCase("$filter=endswith(CustomerId, 'ANATR')", 1)]
+		[TestCase("$filter=indexof(CustomerId, 'ANATR') eq 0", 1, true)]
+		public void StringFilter(string queryString, int expectedCount, bool locateFunction = false)
+		{
+			if(locateFunction && !TestDialect.SupportsLocate)
+				Assert.Ignore("Locate function is not supported.");
```

Comment:
Whitespace glitch (missing space after if), but anyway should be removed after #2392 merging.

Reply:
I trusted Rider for doing such things... Need to check why it's not happening anymore..